style/js/shield_text.js

```diff
@@ -241,15 +242,16 @@ export function drawBannerHaloText(ctx, text, bannerIndex) {
   ctx.font = Gfx.shieldFont(textLayout.fontPx);
   ctx.textBaseline = "top";
   ctx.textAlign = "center";
-  ctx.shadowBlur = 5;
-  ctx.lineWidth = 1.2;
+  ctx.shadowBlur = 0;
+  ctx.lineWidth = 2;
```

Comment:
Let's multiply this by the pixel ratio so it doesn't look too thin at 200% zoom:
![Screenshot from 2022-06-04 15-14-47](https://user-images.githubusercontent.com/1732117/172022405-dac42b8b-3db2-4c06-b727-737d5c318f18.png)
![Screenshot from 2022-06-04 15-14-29](https://user-images.githubusercontent.com/1732117/172022404-3e39b7a7-62df-4703-91d2-d932fcb6f1e5.png)

Reply:
Resolved in 277bfb8 but please verify.